src/tools.js

```diff
@@ -0,0 +1,37 @@
+const formData = require('./index');
+
+let formString = ""; // makes a string to concat with
+if (formData){
+
+
+  // loops through all the objects in the input array
+  formData.forEach( (data)=>{
+    // data is a single input object
+
+    // check on each type of input
+    if(data.type === 'label') {
+      // console.log('it is a', data.type);
+      // add to the formString
+      formString += `<label for='${data.for}'/>`
+
+    } else if (data.type === 'text') {
+      // console.log('it is a', data.type);
+
+      formString += `<input type='${data.type}' name='${data.for}'/>`
+      // add to the formString
+    } else if (data.type === 'submit') {
+      // console.log('it is a', data.type);
+
+      formString += `<input type='${data.type}' name='${data.for}' value='${data.value}'/>`
+      // add to the formString
+    }
+
+
+
+  });
+
+  // Return entire string
+  console.log('the entire string', formString);
+}else{
+  console.log('YOU FUCKED UP');
```

Comment:
Try to remove console logs that don't give helpful feedback. These logs are fine when on your local machine but on a team. This can be offensive, I'm sure no here cares but when you are working for a big company everything you do will have eyes on you. For, example [site tracks bad words in commits on gitHub](http://www.commitlogsfromlastnight.com/)

Reply:
hey mike i did wanted to let you know i did move this it just hasn't updated it thank you